tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>I was going to suggest that we make the structure here the same as the other tutorials with a list in the bullet points above, but since this tutorial uses _a lot_, I quite like this, with mini explanations as to what we're using 🙏 - maybe we can keep this as a standard for more complicated tutorials?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>yeah, I kept it from the Advent challenge because I liked how descriptive it was! </p>